### ISSUE TITLE
calc_pvalue uses old class names

### ISSUE THREAD

**Issue by hpompom** [kvasilopoulos/exuber#17]
---
When I using "calc_pvalue" to run the example, it return this: Error: Argument 'x' should be of class 'radf'

```r
## Not run: 
radf_psy1 <- radf(sim_psy1(100))

calc_pvalue(radf_psy1)
```

**Comment by kvasilopoulos** [comment]
You are right @hpompom `calc_pvalue` has not been adjusted to the new class names of `v0.4.2`, so it doesn't work right now. 

`calc_pvalue` is still experimental and we don't know whether we should keep it in the future. 

If you don't mind, can you let me know why you prefer pvalue over comparing the statistic with the critical values?

Any kind of feedback regarding this function would be very helpful for us.

**Comment by hpompom** [comment]
In my paper, the reviewers thinks me need to check the robust of BSADF result. I think I may be can changes the minium window to calculate the BSADF and then get the mean P-value to prove the result is robust. But I don't know the ideas is reasonable.

**Comment by kvasilopoulos** [comment]
Hi @hpompom, I did a small fix for your issue. If you install the development versions you can run the example now.

```
install.packages("remotes")
remotes::install_github("kvasilopoulos/exuber")
```

Run example:

``` r
library(exuber)
radf_psy1 <- radf(sim_psy1(100))

calc_pvalue(radf_psy1)
#> Using `radf_mc_distr` for `distr`.
#> Warning: ignoring unrecognized snow option(s): progress
#> # A tibble: 1 x 4
#>   id        adf  sadf gsadf
#>   <fct>   <dbl> <dbl> <dbl>
#> 1 series1 0.438 0.051 0.212
```

**Comment by hpompom** [comment]
It can be run but need to using  library(doSNOW). Thank you !

**Comment by hpompom** [comment]
Can it calculate the P-value of BSADF?

**Comment by kvasilopoulos** [comment]
Yeah, there was a problem with doSNOW and doParallel, now I removed doParallel and include doSNOW in the dependencies.

I believe you mean the p-value of the GSADF, right? 

If yes, then it is the 4th column in the data.frame above.

You can easily do it yourself if you want.

* `radf_mc_distr` simulates the distribution for the adf, sadf, and the gsadf test statistic.
* compare the statistic from the `radf` function with the corresponding distribution from `radf_mc_distr`

and you will have the pvalue.

**Comment by hpompom** [comment]
Thank you! The package has helped me a lot. Hope to see more method of detecting bubble in the future!

**Comment by kvasilopoulos** [comment]
Thank you. Happy to hear! We are planning to add a lot more methodologies in the future. Stay tuned!
